allure-web/src/plugins/timeline/styles.css

```diff
@@ -2,40 +2,169 @@
 
 .timeline {
   padding: $gap-size;
+  overflow: auto;
+  height: 100vh;
+
+  &__header {
+    padding-left: 30px;
+    padding-right: 30px;
+    padding-bottom: 5px;
+    text-align: center;
+    color: $app-nav-bg;
+
+    &__min_duration {
+      float:left;
+    }
+
+    &__max_duration {
+      float: right;
+    }
+  }
+
+  &__chart {
+     &__axis_x {
+       .tick {
+         opacity: 0.3;
+         stroke-dasharray: 1,4;
+         shape-rendering: crispEdges;
+
+       }
+     }
+  }
+
+  &__brush {
+     position: absolute;
+     width: 100%;
+     top: 0px;
+
+     .selection {
+       fill: #909090;
+       fill-opacity: 0.8;
+     }
+  }
+
+  &__chart_svg, &__brush_svg {
+    width: 100%;
+  }
 
   &__item:hover {
     opacity: 0.7;
   }
 
   &__host {
     font-size: 12px;
+    fill: #838383;
+    dominant-baseline: central;
   }
 
   &__host-bg {
-    fill: none;
-    stroke: $border-accent-color;
-    shape-rendering: crispEdges;
-  }
+    fill: #eee;
+    stroke: #e0e0e0;
 
-  &__item {
-    shape-rendering: crispEdges;
+    rx:3;
+    ry:9;
   }
 
   &__left_handle, &__right_handle {
     font-family: 'FontAwesome';
     font-size: 10px;
+    fill: #808080;
     dominant-baseline: central;
-    fill: steelblue;
-    opacity: .15;
+
+    opacity: 1;
     cursor: default;
+    text: 'f';
   }
 
   &__left_handle {
     text-anchor: end;
   }
+
 }
 
-.selection {
-  fill: steelblue;
-  fill-opacity: 0.15;
+input[type=range] {
```

Comment:
These styles will be applied to **all** inputs that we have in Allure face.
Use class name selector `.timeline__range_bar`